src/status_im/contexts/chat/home/add_new_contact/events.cljs

```diff
@@ -1,13 +1,12 @@
 (ns status-im.contexts.chat.home.add-new-contact.events
   (:require
     [clojure.string :as string]
+    [re-frame.core :as re-frame]
```

Comment:
I'm always unsure when we want to use `utils.re-frame` and `re-frame.core`. 
@ilmotta, @flexsurfer is there any reasoning we have both of these options when a lot of the same logic is accessible from both??